src/App.tsx

```diff
@@ -1,61 +1,120 @@
 import './App.scss';
+import { ChangeEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function getUser(userId: number): User | null {
```

Comment:
don't forget about past checklists 
```suggestion
function getUserById(id: number): User | null {
```